spec/features/switch_organisation_spec.rb

```diff
@@ -1,31 +1,59 @@
 require 'rack_session_access/capybara'
 
 describe 'Multiple organisations', type: :feature do
-  let!(:other_organisation) { create(:organisation) }
-  let(:user) { create(:user, :with_organisation) }
-  let(:organisation) { user.organisations.first }
+  let(:organisation_1) { create(:organisation) }
+  let(:organisation_2) { create(:organisation) }
+  let(:user) { create(:user) }
 
   before do
+    user.organisations << [organisation_1]
```

Comment:
Usually << an item into an array.  Unsure how this is working.

user.organisations << organisation_1